Assets/ConsentManagementProvider/Scripts/json/JsonUnwrapper.cs

```diff
@@ -271,8 +271,8 @@ private static GdprConsent UnwrapGdprConsent(GdprConsentWrapper wrapped)
 
                     var vendorGrantValue = JToken.FromObject(vendorGrantWrapper.Value);
 
-                    if (vendorGrantValue["granted"] != null)
-                        isGranted = vendorGrantValue["granted"].ToObject<bool>();
+                    if (vendorGrantValue["vendorGrant"] != null)
+                        isGranted = vendorGrantValue["vendorGrant"].ToObject<bool>();
```

Comment:
@OmarVector let me check if it interferes with Android implementation.

Reply:
@OmarVector I confirm, that's true. Including this PR to the closest release.

Reply:
@OmarVector you can merge when ready; we're going to include this to release 2.3.2 today anyway

Reply:
@Nevazhnovu 
Thank you:)

I dont have write access to merge it to your branch:)
however I merged with my forked release one:))

Thanks once again for your patience:))

Reply:
You welcome! :)